### ISSUE TITLE
Change column names to avoid using reserved words

### ISSUE THREAD

**Issue by smclay** [ICRAR/ngas#23]
In Oracle "comment" is a reserved word. Could we change the `ngas_subscr_queue.comment` table column to a new name in a future release?
Also, perhaps changing `ngas_disks.archive` and `ngas_disks.type` would also be a good idea.

**Comment by rtobar** [comment]
Yes, there are, and there have been in the past, such conflicts with keywords (the one I'm aware of is "ignore", I think it's a keyword in MySQL). Probably changing the names in the schema and then in the code is fairly easy, but if one wants to ensure backwards compatibility (e.g., if you want to rollback to an old NGAS version, or use old/new versions concurrently) this might considerably harder.

One alternative would be to do something similar to what we did with the `ignore` v/s `file_ignore` column name in `ngas_files`, where the code is flexible to use either based on a configuration value (so it's up to users which one to use). A different alternative would be to move slowly towards keyword-"safe" column names, with one release of NGAS adding the new columns and writing data to both new and old (again, to ensure backwards-compatibility), then at a later release removing the old column and sticking with the new one.

A third alternative (what we are doing) is enclosing "comment" in double-quotes, which seems to work in all engines, but is of course more tedious for those having to write queries by hand.
